docs/private-ddn/ddn-workspace.mdx

```diff
@@ -0,0 +1,152 @@
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in the VS Code editor interface.
+
+## Step 4. Add a data source
+
+The command below will launch a wizard via the DDN CLI which will assist you in adding a data connector. You can learn
+more about adding sources via data connectors [here](/data-sources/connect-to-a-source.mdx).
+
+```sh title="Add a data source using the CLI:"
+ddn connector init <connector_name> -i
+```
+
+This will provision the necessary files — using your configuration — for connecting a data source to your supergraph
+API.
+
+:::info Organize your data into subgraphs
+
+You can also organize your API into **subgraphs**. Subgraphs are generally organized around team functions or business
+outcomes and allow for independent ownership and governance of data sources. Learn more
+[here](/project-configuration/subgraphs/index.mdx).
+
+:::
+
+## Step 5. Generate your metadata
+
+```sh title="First, introspect your data source:"
+ddn connector introspect <connector_name>
+```
+
+```sh title="Then, generate the metadata:"
+ddn model add <connector_name> '*'
+ddn command add <connector_name> '*'
+ddn relationship add <connector_name> '*'
+```
+
+You'll see metadata files generated for each entity in your data source. You can learn more
+[here](/data-modeling/overview.mdx).
+
+:::info Note
+
+Local Development and launching the local console is not supported in DDN workspace. Once the metadata is generated, create a cloud project and deploy your supergraph to your Private DDN to query the API.
+
+:::
+
+## Step 6. Create a project on your Data Plane
+
+You'll create a new project using the DDN CLI and by passing two flags: the `data-plane-id` and the `plan`.
+
+:::info Get Data Plane ID
+
+You can find all the Data Planes you have access to on this page: https://console.hasura.io/data-plane
+
+:::
+
+:::info Available Plans
+
+- ddn_free
+- ddn_base
+- ddn_advanced
+
+Read more about plans [here](/reference/pricing.mdx)
+
+:::
+
+```sh title="Create a new project using your Data Plane ID and plan choice:"
+ddn project init --data-plane-id <data-plane-id> --plan <plan-name>
+```
+
+The CLI will return information about your newly created project.
+
+## Step 7. Build and deploy your supergraph
+
+```sh title="build and deploy you supergraph:"
+ddn supergraph build create
+```
+
+When this process is complete, the CLI will return a link to the hosted API where you can query your data.
+
+## Step 8. Exporting your metadata
+
+You can export the metadata to your local machine by right clicking on the `Explorer` pane and selecting on the `Download` option , by. This will download the entire directory containing all the files in your workspace. Workspace also has `git` pre-installed, so you can configure it and push your changes to a remote repository as well. 
```

Comment:
```suggestion
You can export the metadata to your local machine by right clicking on the `Explorer` pane and selecting on the `Download` option. This will download the entire directory containing all the files in your workspace. Workspace also has `git` pre-installed, so you can configure it and push your changes to a remote repository as well. 
```